src/ert/gui/ertwidgets/ensembleselector.py

```diff
@@ -95,7 +115,10 @@ def _ensemble_list(self) -> Iterable[Ensemble]:
             )
         else:
             ensembles = self.notifier.storage.ensembles
-        ensemble_list = list(ensembles)
+        if self._show_only_with_valid_experiment:
+            ensemble_list = [ens for ens in ensembles if ens.experiment.is_valid()]
```

Comment:
We probably want to show ensembles in the ensemble selector if it is invalid, but present it as disabled or bad in some way.